jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyRunMojo.java

```diff
@@ -544,9 +546,6 @@ public void restartWebApp(boolean reconfigureScanner) throws Exception
                 continue;
             }
 
-            if (Artifact.SCOPE_PROVIDED.equals(artifact.getScope()))
-                continue; //never add dependencies of scope=provided to the webapp's classpath (see also <useProvidedScope> param)
-
             if (Artifact.SCOPE_TEST.equals(artifact.getScope()) && !useTestScope)
```

Comment:
Maybe move this line up to the same place as the Artifact.SCOPE_PROVIDED test.

Reply:
@janbartel good catch!